### PR TITLE
Fixed Webpack Cache Bug

### PR DESCRIPTION
**Pull request by eddieferrer**
* Fixes Could not freeze FILE_NAME: Cannot read property 'hash' of undefined bug/lines in log once and for all

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -145,7 +146,7 @@
 		"file-loader": "^6.0.0",
 		"friendly-errors-webpack-plugin": "^1.7.0",
 		"git-revision-webpack-plugin": "^3.0.3",
-		"hard-source-webpack-plugin": "^0.13.1",
+		"hard-source-webpack-plugin-fixed-hashbug": "^0.13.2",
```
Comment (mcstover) [inline package.json]: lol

Comment (eddieferrer) [reply package.json]: Its a fork, someone in the community got tired of the PR sitting around with the fix for 2 years....

File: package.json [hunk 2]
```diff
@@ -45,7 +45,6 @@
 		"algoliasearch": "^3.34.0",
 		"apollo-cache-inmemory": "^1.6.5",
 		"apollo-client": "2.6.8",
-		"apollo-fetch": "^0.7.0",
```
Comment (mcstover) [inline package.json]: were you previously using this for the Contentful stuff?

Comment (eddieferrer) [reply package.json]: Yeah, I've been testing this PR for a couple of days, while working on other things and that snuck in there, removed from this PR now

**Pull request merged**